src/components/MoviesList.js

```diff
@@ -2,21 +2,31 @@ import React, {useEffect, useState} from 'react'
 import { Card, Col, Row } from 'react-bootstrap';
 import { useSelector } from 'react-redux'
 import { Link } from 'react-router-dom';
-import { getAllMovies } from '../features/movies/movieSlice'
+import {movieSelectors} from '../features/movies/movieSlice'
+import { useDispatch } from 'react-redux';
+import { fetchAllMovies } from '../features/imdb.service';
 import './movieList.css';
 
 function MoviesList() {
     const [screenSize, setScreenSize] = useState(null);
+    let movies = useSelector(movieSelectors.selectAll);

```

Comment:
In `movie.js` we are using `useSelector(state => ...)` syntax and here we are using selectors defined in slice. I recommend the first option, it gives more flexibility. Let's use that through-out the code base.